apps/mobile-alphabet/sample.theme.json

```diff
@@ -0,0 +1,18 @@
+{
+    "colors": {
```

Comment:
You might want to have a type for this. Either make this a `ts` file or have a utility that reads the json and builds a proper `ts` object where you validate the properties. Otherwise, a missing property in the git-ignored json file could cause the build to blow up at run time. You can add a separate story for this.